### DISCUSSION TITLE
How to save -debug-req results?

### DISCUSSION THREAD

**Discussion by dodoaska** [projectdiscovery/nuclei#1141] [Q&A] [answered]
how i save -debug-req ?

C:\Users\Administrator>nuclei -list C:\Users\Administrator\Desktop\excell.txt -t C:\Users\Administrator\Downloads\nuclei-templates-master\cves\2021\CVE-2021-xxx.yaml -o C:\Users\Administrator\Desktop\vul.txt -debug-req

**Comment by ehsandeep** [comment, marked as answer]
```
C:\Users\Administrator>nuclei -list C:\Users\Administrator\Desktop\excell.txt -t C:\Users\Administrator\Downloads\nuclei-templates-master\cves\2021\CVE-2021-xxx.yaml -o C:\Users\Administrator\Desktop\vul.txt -debug-req 2> output.txt
```

This will save debug logs + results to **output.txt**, nothing will be printed in CLI / terminal.

**Reply by dodoaska** [reply]
Thank you
